platform/uwp/os_uwp.cpp

```diff
@@ -452,29 +452,32 @@ void OS_UWP::ManagedType::on_accelerometer_reading_changed(Accelerometer ^ sende
 	AccelerometerReading ^ reading = args->Reading;
 
 	os->input->set_accelerometer(Vector3(
-			reading->AccelerationX,
-			reading->AccelerationY,
-			reading->AccelerationZ));
+										 reading->AccelerationX,
```

Comment:
If `clang-format` is forcing this, I'd argue that it would be more readable to keep them all on the same line as `Vector3(`.

Reply:
Yeah unfortunately the clang-format CI build step was forcing me that formatting... I'll try and make the Vector3 in one line.